selinux/ipa.te

```diff
@@ -412,3 +420,28 @@ optional_policy(`
 optional_policy(`
        systemd_private_tmp(ipa_custodia_tmp_t)
 ')
+
+optional_policy(`
+    gen_require(`
+        type tomcat_t;
+    ')
+    can_exec(tomcat_t, ipa_pki_retrieve_key_exec_t)
+    pki_manage_tomcat_etc_rw(ipa_pki_retrieve_key_t)
+')
+
+optional_policy(`
+    gen_require(`
+        type devlog_t;
+    ')
+
+    dontaudit ipa_custodia_t devlog_t:lnk_file read_lnk_file_perms;
```

Comment:
A new interface in the selinux-policy needs to be created for this permission, too.

Reply:
Would it be acceptable to keep it as is?

Reply:
@zpytela O.k., what do you suggest to do now with this? Keep or remove?

Reply:
You can now remove this whole optional block - it should be obsoleted by the `logging_send_syslog_msg(ipa_custodia_t)`.